src/development/add-to-app/index.md

```diff
@@ -16,24 +16,12 @@ app's UI in Flutter. Or, just to run shared Dart logic.
 In a few steps, you can bring the productivity and the expressiveness of
 Flutter into your own app.
 
-As of Flutter v1.12, add-to-app is supported for the basic scenario of
-integrating one full-screen Flutter instance at a time per app.
-It currently has the _**following limitations**_:
-
-* Packing multiple Flutter libraries into an
-  application isn't supported.
-* Plugins used in add-to-app on Android should migrate
-  to the [new Android plugin APIs][Android plugin APIs], based on [`FlutterPlugin`].
-* Plugins that don't support `FlutterPlugin` might have unexpected
-  behaviors if they make assumptions that are untenable in add-to-app
-  (such as assuming that a Flutter `Activity` is always present).
-* As of v1.17, the Flutter module only supports AndroidX applications on Android.
-
-As of Flutter v1.26, add-to-app experimentally supports adding multiple
-instances of Flutter engines, screens, or views into your app. This can
-help integration scenarios such as a hybrid navigation stack with mixed
+add-to-app supports integrating multiple instances of any screen size.
```

Comment:
Starting a paragraph with a lower case letter feels odd. Maybe wrap `add-to-app` in backticks?

/cc @sfshaza2